docs_input/build.rst

```diff
@@ -120,3 +134,56 @@ to browse the documentation. Note that the most recent version of the documentat
 
 https://nvidia.github.io/MatX/
 
+MatX in Offline Environments
```

Comment:
I would put this section towards the bottom since it's likely not as useful to the public.